aws-cloudwatch/README.md

```diff
@@ -0,0 +1,135 @@
+1. Go to the AWS searchbar, and search for IAM service
+1. Go to the `Policies` section. and click `Create policy`
+1. Remove the `Popular services` flag, and select `CloudWatch` service. 
+1. Now, select `GetMetricData`, `PutMetricData`, `ListMetrics` actions and click `Next`
+1. Enter the policy name and click `Create policy`
+
+Now, we should create the policy for CloudWatch Logs:
+1. Repeat the first two steps you did while creating the policy for metrics
+1. Remove the `Popular services` flag, and select `CloudWatch Logs` service.
+1. Now, select `CreateLogGroup`, `CreateLogStream`, `PutLogEvents` actions and click `Next`
+1. Specify resource ARN for selected actions
+1. Enter the policy name and click `Create policy` 
+
+After creating the IAM Policies, we can finally create an IAM User:
+1. Go to the `Users` section and click `Add user`
+1. Enter the user name and click next
+1. Click to the `Attach policies directly`
+1. Select two new policies you added on previous steps as well as the `AWSXrayWriteOnlyAccess`
+1. Click `Next` and then `Create User`
+1. Open the new user
+1. Go to the `Security credentials` tab and click `Create access key`
+1. Select `Application running outside AWS` and then click `Next`
+1. Describe the purpose of this access key and click `Create access key`
+1. Now copy and save `Access key` and `Secret access key`
+
+### Create a secret with AWS Credentials
+
+In order to connect OTEL Collector to AWS we need to define security credentials in the kyma system. 
+
+1. In the [values.yaml](./aws-secret/values.yaml), replace the `{ACCESS_KEY}` and `{SECRET_ACCESS_KEY}` to your access keys, and `{AWS_REGION}` with the AWS region you want to use
+2. Now, create the secret by using 
+    ```bash
+    kubectl apply -f ./aws-secret/aws-secret.yaml
+    ```
+
+### Deploy an OTEL Collector
+
+After creating a secret and configuering AWS, we can finally deploy an Otel Collector itself.
+
+1. Deploy an OTEL Collector by calling 
+    ```bash
+    kubectl -n $KYMA_NS apply -f ./aws-otel-collector/aws-otel.yaml
+    ```
+
+### Create pipelines
+
+After deploying OTEL Collector itself, you should deploy logpipeline, tracepipeline, and metricpipeline. 
+
+1. Deploy a logpipeline by calling 
+    ```bash
+    kubectl apply -f ./pipelines/logpipeline.yaml
+    ```
+1. Replace `{NAMESPACE}` and deploy a tracepipeline by calling 
+    ```bash
+    kubectl apply -f ./pipelines/tracepipeline.yaml
+    ```
+1. Replace `{NAMESPACE}` and deploy a metricpipeline by calling 
+    ```bash
+    kubectl apply -f ./pipelines/metricpipeline.yaml
+    ```
+
+## Verifying the results by deploying sample apps
+
+In order to verify the results of CloudWatch and X-Ray we will deploy sample applications for each service accordingly.
+
+### Verifying CloudWatch traces, logs, and metrics arrival 
+
+In order to deploy sample app which generates traces that we took from [aws-otel tutorial](https://docs.aws.amazon.com/eks/latest/userguide/sample-app.html):
```

Comment:
Why we cannot deploy it directly from there? Do not duplicate the manifests without reason as we would need to maintain it